hotfixes/4.3.15.cfg

```diff
@@ -0,0 +1,12 @@
+[buildout]
+
+hotfix-eggs =
+
```

Comment:
When running `hotfixes/update.py`, there are hotfixes added:
```diff
+    Products.PloneHotfix20160830
+    Products.PloneHotfix20161129
+    Products.PloneHotfix20170117
```

I'm not sure though whether those hotfixes still apply to 4.3.15; an update in http://jone.github.io/plone-hotfixes/ may be necessary 🤔 

⌛️ I'm gonna update the plone-hotfixes constraints, so that we can run the `update.py`.